Provider/src/main/java/com/spotify/confidence/ConfidenceFeatureProvider.kt

```diff
@@ -244,89 +195,92 @@ class ConfidenceFeatureProvider private constructor(
         fun isStorageEmpty(
             context: Context
         ): Boolean {
-            val storage = StorageFileCache.create(context)
+            val storage = FileDiskStorage.create(context)
             val data = storage.read()
-            return data == null
+            return data == FlagResolution.EMPTY
         }
 
         @Suppress("LongParameterList")
         fun create(
```

Comment:
Can we have a deprecated, backwards compatbile `create` fun that works as before with the `clientSecret` and `region`?